main.py

```
@@ -520,6 +585,11 @@ async def handle_message_sent(self, event: AstrMessageEvent):
         """
         chat_id = event.unified_msg_origin
         try:
+            if self._is_whitelist_blocked(chat_id):
+                logger.debug(
+                    f"AngelHeart[{chat_id}]: 会话未在白名单中，跳过发送后处理。"
```

Ruff (0.16.1)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)